PantryPal/App.js

```diff
@@ -1,14 +1,47 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { FlatList, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+
+const list = [
+  {
+    product: 'Item #1',
+    quantity: 1,
+    forShopping: true,
+  },
+  {
+    product: 'Item #2',
+    quantity: 2,
+    forShopping: false,
+  },
+]
 
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>YOU DID IT, welcome to PANTRY PAL</Text>
-      <Text>Open up App.js to start working on your app!</Text>
+      <FlatList
+        data={list}
+        renderItem={({ item }) => <Text style={styles.item}>{item.product}</Text>} // Needs ListItem
+      />
+
+
+      <TouchableOpacity
```

Comment:
Not really clear from a glance what this is - maybe comment that this is the plus button